lib/utils.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const LinkHeader = require('http-link-header');
+
+const getCrossOrigin = (url = '', host) => {
+    if (host && url.indexOf(`://${host}`) < 0) {
```

Comment:
`url.includes(...)` is a tiny bit more expressive

Reply:
done